### DISCUSSION TITLE
mzTab for DIA-NN

### DISCUSSION THREAD

**Discussion by wanghe98** [bigbio/quantms#317] [General]
Hi, sorry for bothering you.

May I ask could the reports from DIA-NN be converted to mzTab or mzIdentML formats for complete submission in PRIDE?

Thanks!

**Comment by ypriverol** [comment]
Dear @wanghe98 the current DIA workflow based on DIA-NN exports the data to mzTab which is perfectly compatible with ProteomeXchange Submissions.

**Reply by wanghe98** [reply]
Thanks very much for your fast reply! Even on weekends!
